community/working-groups.md

```diff
@@ -0,0 +1,57 @@
+## Working Groups
+
+A working group is a group of people that are interested in working on a common
+topic. Working groups are informal and have no official authority. All working
+groups have a public channel on the [Julia Slack][slack]. Many working groups
+also hold regular meetings that are held online and are open to anyone. Meeting
+links are posted in the relevant Slack channels.
+
+The working group meetings are also listed on the
+[community events calendar][commmunity-events-calendar].
+
+### List of working groups
+
+Note: "ET" refers to Eastern Time.
+
+| Working Group                    | Slack channel       | Meetings                                 | Contact                                                         |
+| -------------------------------- | ------------------- | ---------------------------------------- | --------------------------------------------------------------- |
+| Base Julia CI                    | `#ci-dev`           | Every other Tuesday at 2:15pm ET         | Elliot Saba (@staticfloat), Dilum Aluthge (@DilumAluthge)       |
+| [GPU][gpu]                       | `#gpu`              | Every other Monday at 11am ET            | Tim Besard (@maleadt), Valentin Churavy (@vchuravy)             |
+| High-performance computing (HPC) | `#hpc`              | Fourth Tuesday of each month at 2pm ET   | Valentin Churavy                                                |
```

Comment:
```suggestion
| High-performance computing (HPC) | `#hpc`              | Fourth Tuesday of each month at 2pm ET   | Valentin Churavy (@vchuravy)                                               |
```

Reply:
Oh I didnt see that it was in the previous listing

Reply:
Could make them name a link to the user instead to save space.

Reply:
The link idea sounds good.